src/components/Clothing.js

```diff
@@ -32,10 +32,10 @@ function Clothing({
       {showDetails && (
         <StyledWrapper>
           <h2>{title}</h2>
-          <p>Farbe: {color}</p>
-          <p>Muster: {pattern}</p>
-          <p>Material: {material}</p>
-          <p>Passform: {fitting}</p>
+          <StyledP>Farbe: {color}</StyledP>
+          <StyledP>Muster: {pattern}</StyledP>
+          <StyledP>Material: {material}</StyledP>
+          <StyledP>Passform: {fitting}</StyledP>
```

Comment:
I would probably leave it as `p` and then style all `p` elements within the `StyledWrapper` styles.